Assets/Scripts/Runtime/Character/Minion.cs

```diff
@@ -11,21 +12,89 @@ public class Minion : Creature
 
         private const float STOPPING_DISTANCE = 0.5f;
 
+        private static int s_spawned_count = 1;
```

Comment:
a spawned counter here?

Reply:
I don't understand

Count is a proper noun
https://www.merriam-webster.com/dictionary/count
![image](https://github.com/user-attachments/assets/8e290493-8a3f-4921-98f8-0c0b6ab3f6d0)

Is this some other suggestion than this variable name?

Reply:
A mistake